EC-CUBE/ec-cube
# 【ページ管理】検索窓があるが機能していない

## 概要(Overview・Refs Issue)
   【ページ管理】検索窓があるが機能していない

## 方針(Policy)

## 実装に関する補足(Appendix)

## テスト（Test)
- chrome/firefox

Case 1:
Steps:
1. Login Admin page
2. Go to nav: コンテンツ管理 -> ページ管理
3. Search: Enter some text at the input - top right
Expected result: Show the corresponding pages
